src/main/java/com/epam/esm/controller/CertificateController.java

```diff
@@ -74,32 +61,27 @@ public void deleteById(@PathVariable long id) {
      * Create new GiftCertificate
      * and save it to the db with all tags which was linked to the certificate in certificateDTO
      *
-     * @param certificateDTO - dto with certificate and linked tags
+     * @param certificate - certificate and linked tags
```

Comment:
Why you've changed this? if your param named certificateDTO please leave here certificateDTO, or rename parameter